(DOCSP-12824) Manually Deleting an Object With A Relationship [Node/React Native] (#607)
mongodb/docs-realm: source/node/writes.txt

      :language: javascript
      :emphasize-lines: 6

.. _node-chaining-delete:

Manually Delete An Object with a Relationship
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When you delete an object with a :ref:`relationship <node-client-relationships>`
> to another object, Realm does not automatically delete the other object. To
> delete an object with a relationship to another object, you must manually delete
> the object.

Reviewer: The wording of this is very confusing, imo. I had to check the JIRA ticket to fully understand the issue being resolved.
```suggestion
When you delete a non-embedded object with a :ref:`relationship <node-client-relationships>`
to a child object, Realm does not automatically delete the child object.
```

Author: That does sound better. Taking the suggestion. Also, adding it to the react native doc as well.
